### DISCUSSION TITLE
The window is severely deformed when switching back from the background

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu-framework#5406] [Q&A] [answered]
win+D minimize window and switch back
problem appear
Switching back to the window again will return to normal
But the cursor will be limited to the upper left corner
I am playing on windows 11
![image](https://user-images.githubusercontent.com/74862110/190079100-51501849-a54b-4177-b3eb-f6bb5861532c.png)
[updater.log](https://github.com/ppy/osu/files/9563543/updater.log)
[runtime.log](https://github.com/ppy/osu/files/9563545/runtime.log)
[network.log](https://github.com/ppy/osu/files/9563546/network.log)
[performance.log](https://github.com/ppy/osu/files/9563547/performance.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu-framework/issues/5407
